packages/nextjs/app/builders/0xF11046328814004583c8e74050f0d59520f53e90/page.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+interface ProfileLink {
+  name: string;
+  url: string;
+}
+
+const profileLinks: ProfileLink[] = [
+  { name: "GitHub", url: "https://github.com/Welith" },
+  { name: "Twitter", url: "https://twitter.com/bo4ka7a" }, // Replace with URL
+];
+
+const Home: React.FC = () => {
+  return (
+    <div style={{ textAlign: "center", paddingTop: "50px" }}>
+      <header>
+        <h1>Boris Kolev</h1>
+        <p>Passionate about technology</p>
+      </header>
+      <section style={{ marginBottom: "2rem" }}>
+        <img
```

Comment:
This is causing a linting issue: https://github.com/BuidlGuidl/batch4.buidlguidl.com/actions/runs/8620398825/job/23654029724?pr=17

![image](https://github.com/BuidlGuidl/batch4.buidlguidl.com/assets/2486142/9f31839a-15c3-460e-95d9-4e4116a78829)

---

Check https://github.com/BuidlGuidl/batch4.buidlguidl.com/pull/13#discussion_r1557610727